AWESOME_LIST.md

```diff
@@ -0,0 +1,19 @@
+# Awesome List
```

Comment:
Thank you, @caraya10 ! Do you mind also adding IBA to the list ? https://github.com/pytorch/captum/issues/358